requirements.txt

```diff
@@ -6,4 +6,11 @@ balance
 jupytext
 fastexcel
 tqdm
-sklearn
\ No newline at end of file
+sklearn
```

Comment:
This should be `scikit-learn`, `pip` throws an error with `sklearn` as this is outdated.

Reply:
Also needs `regex`

Reply:
Also needs `pyogrio`

Reply:
Ah great thank you for flagging all these!

I thought regex is a built-in module with python. Did you need to install it separately?

Reply:
Ignore `regex`, that seems to have been because I failed to import properly when testing.